src/web/parserUtils.ts

```diff
@@ -0,0 +1,208 @@
+import type * as MarkdownTypes from '../../types/types';
+import * as CursorUtils from './cursorUtils';
+import type * as StyleUtilsTypes from '../styleUtils';
+
+type PartialMarkdownStyle = StyleUtilsTypes.PartialMarkdownStyle;
+type MarkdownType = MarkdownTypes.MarkdownType;
+type MarkdownRange = MarkdownTypes.MarkdownRange;
+
+type NestedNode = {
+  node: HTMLElement;
+  endIndex: number;
+};
+
+function addStyling(targetElement: HTMLElement, type: MarkdownType, markdownStyle: PartialMarkdownStyle) {
+  const node = targetElement;
+  switch (type) {
+    case 'syntax':
+      Object.assign(node.style, markdownStyle.syntax);
+      break;
+    case 'bold':
+      node.style.fontWeight = 'bold';
+      break;
+    case 'italic':
+      node.style.fontStyle = 'italic';
+      break;
+    case 'strikethrough':
+      node.style.textDecoration = 'line-through';
+      break;
+    case 'mention-here':
+      Object.assign(node.style, markdownStyle.mentionHere);
+      break;
+    case 'mention-user':
+      Object.assign(node.style, markdownStyle.mentionUser);
+      break;
+    case 'link':
+      Object.assign(node.style, {
+        ...markdownStyle.link,
+        textDecoration: 'underline',
+      });
+      break;
+    case 'code':
+      Object.assign(node.style, {
+        ...markdownStyle.code,
+        fontFamily: 'monospace',
+      });
```

Comment:
`markdownStyle.code` already includes `fontFamily` with the default value of `monospace` on Web.
```suggestion
      Object.assign(node.style, markdownStyle.code);
```